module/VuFind/src/VuFind/Auth/AbstractBase.php

```diff
@@ -267,6 +267,17 @@ public function updatePassword($request)
         );
     }
 
+    /**
+     * Check if session initiator is used.
+     *
+     * @return bool
+     */
+    public function hasSessionInitiator(): bool
+    {
+        $reflector = new \ReflectionMethod($this, 'getSessionInitiator');
```

Comment:
This makes sense to me, though I feel a little strange about using reflection to solve this problem. Would it be less expensive to simply `return (bool)$this->getSessionInitiator('dummy-value');` ?